docs/user-documentation/media.md

```diff
@@ -1,60 +1,79 @@
 Drupal 8 recognizes files (such as images, audio files, video files, etc.) but wraps each file in an intermediate structure called a
-"media" to allow us to attach fields to files. It is in a media's fields that we store information about the media's file, such as file
+"media" to allow us to attach fields to files. Drupal uses a media's fields to store information about the media's file, such as file
 size, width and height (for images), alt text (for images), creation date, and so on.
 
 !!! note "Compared to Islandora 7"
     In Islandora 7, this sort of technical metadata would have been stored in a single RELS-INT datastream.
```

Comment:
Not entirely true. We manually put height and width of images there, but that was a bit of a bastardization of RELS-INT which was supposed to relate individual datastreams _to each other_. Some stuff, like file size, mimetype, and creation date, Fedora considered to be properties of the datastream. Here are some docs about how Tuque handles them: https://github.com/Islandora/islandora/wiki/Working-With-Fedora-Objects-Programmatically-Via-Tuque#working-with-datastreams

Reply:
Okay. This is a pre-existing note that I simply left. I didn't touch it since I have *no* experience with 7.x.

Would a better phrasing be "this sort of technical metadata would have been stored as datastream properties or as additional metadata-specific datastreams"?

Reply:
sure.